core/node/groups.go

```diff
@@ -122,6 +122,12 @@ func LibP2P(bcfg *BuildCfg, cfg *config.Config) fx.Option {
 			logger.Error("Use the `Swarm.Transports.Network.Relay' config field instead")
 		}
 	}
+	//nolint
+	if cfg.Swarm.EnableRelayHop {
+		logger.Error("The `Swarm.EnableRelayHop` config field is deprecated.\n" +
+			"Use `Swarm.RelayService` to configure the circuit v2 relay.\n" +
+			"If you want to continue running a circuit v1 relay, please use the standalone relay daemon: https://github.com/libp2p/go-libp2p-relay-daemon\n")
```

Comment:
@marten-seemann  can relay daemon run both at the same time, or do we also need to specify `Config.RelayV2.Enabled: false` if we want V1?
```suggestion
			"If you want to continue running a circuit v1 relay, please use the standalone relay daemon: https://github.com/libp2p/go-libp2p-relay-daemon (with Config.RelayV1.Enabled: true)\n")
```

Reply:
relayd can run both at the same time.